content/doc/book/system-administration/administering-jenkins-on-kubernetes.adoc

```diff
@@ -12,4 +12,163 @@ ifdef::backend-html5[]
+
+Jenkins X at the moment supports kubernetes versions 1.20 and 1.21. Version 1.22 introduces a lot of deprecations, and is not fully supported yet. In the future we aim to support the same set of versions supported by the major cloud providers, Google, Amazon and Azure. These versions are described in the links below, ranging from version 1.20 to version 1.24.
+
+* link:https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html[Amazon]
+* link:https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#aks-kubernetes-release-calendar[Azure]
+* link:https://cloud.google.com/kubernetes-engine/docs/release-notes[Google]
+
+*Upgrade Procedure:*
+
+* It is highly recommended to test upgrades on a separate cluster, where you have all the same setup as you use in production.
+
+* Run link:https://github.com/FairwindsOps/pluto[Pluto] to find any deprecated CRDs present in your cluster. You can run Pluto both on the cluster using 'pluto detect-helm', and on the files on your cluster repository using 'pluto detect-files'.
+
+* Cross-check the list of deprecations you find, with the list from link:https://kubernetes.io/docs/reference/using-api/deprecation-guide[Kubernetes] and see what is required for the Kubernetes version you want to upgrade to.
+
+* Fix what needs to be fixed, and deploy the changes.
+
+* You may now start the upgrade of Kubernetes.
+** The process has two steps, first, you need to upgrade the control plane, and next upgrade the nodes. Follow the upgrade steps according to your cloud provider.
+*** link:https://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html[Amazon]
+*** link:https://learn.microsoft.com/en-us/azure/aks/upgrade-cluster?tabs=azure-cli[Azure]
+*** link:https://cloud.google.com/kubernetes-engine/docs/how-to/upgrading-a-cluster[Google]
+
+* Pour yourself a cup of your favorite drink while you sit back and wait for the magic to happen.
+
+=== Upgrade Kubernetes Cluster maintained by the Jenkins Infrastructure project:
+
+*Requirements:*
+
+* Create link:https://issues.jenkins.io[Jira] Ticket.
+
+* Announce on:
+** link:https://github.com/jenkins-infra/status[Jenkins Status] page.
+** IRC, channel #jenkins-infra on freenode.
+** link:https://groups.google.com/g/jenkins-infra[Jenkins Infrastructure and Dev Mailing list]
+** link:https://twitter.com/jenkinsci/[Twitter], in case of major upgrade.
+
+*  Review Release Changelogs:
+** link:https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG[Kubernetes]
+** link:https://github.com/Azure/AKS/blob/master/CHANGELOG.md[AKS]
+
+* Review and merge important pull request from link:https://github.com/jenkins-infra/charts[jenkins-infra/charts].
+
+* Block k8smgmnt job on infra.ci
```

Comment:
```suggestion
Review Release Changelogs:
* link:https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG[Kubernetes]
* link:https://github.com/Azure/AKS/blob/master/CHANGELOG.md[AKS]

Review and merge important pull request from link:https://github.com/jenkins-infra/charts[jenkins-infra/charts].

Block k8smgmnt job on infra.ci
```
If the * are intended to help create a list, would using the "." to create an ordered/numbered list work as well? This would provide a better idea of if these are steps/instructions or best practices. Since the process is upgrading Kubernetes, maybe it should be numbered steps? The "*" can still be used to denote secondary items in the numbered list, but just using the * may not display properly in the guide.